boot/bootutil/include/bootutil/image.h

```diff
@@ -52,6 +47,7 @@ struct flash_area;
 /*
  * Image trailer TLV types.
  */
+#define IMAGE_TLV_KEYHASH           0x01   /* hash of the public key */
```

Comment:
Same thing here; re-using old values given that 0.9 was a release feels like asking for trouble. Why does this have to be number one?